### PR TITLE
Added overture arranger

### PR DESCRIPTION
**Pull request by plaplante-lab**

### CODE REVIEW

File: arranger/configmaps.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+data:
+  ES_HOST: http://elasticsearch-workers:9200/
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: arranger-admin-ui-config
+data:
+  HOST: http://arranger-server.qa.clin.ferlab.bio
+  PORT: 5050
```
Comment (Magnitus-) [inline arranger/configmaps.yml]: You don't need to put "config" in the names. Kubernetes namespaces resource names by resource type, so there will not be conflict with other resources that have the same name.

File: arranger/configmaps.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  name: arranger-server-config
+data:
+  ES_HOST: http://elasticsearch-workers:9200/
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: arranger-admin-ui-config
+data:
+  HOST: http://arranger-server.qa.clin.ferlab.bio
```
Comment (Magnitus-) [inline arranger/configmaps.yml]: This url is very environment-specific. I'd leave it with the value "\_\_UNDEFINED\_\_" and overwrite it with a mergepatch in cqdg-environments.

File: arranger/namespace.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: overture
```
Comment (Magnitus-) [inline arranger/namespace.yml]: Adding a namespace is good, but then you need to add a "namespace: overture" metadata entry for all your resources.

Alternatively, for a slightly DRYer experience, you can add a "namespace" entry in your kustomization file and kustomize will take care of adding a "namespace" metadata field to all the resources for you: https://kubernetes-sigs.github.io/kustomize/api-reference/kustomization/namespace/

However, note that there are some limitations to namespaces:
- You can't use secrets and configmaps across namespaces
- Reference to services in other namespaces need to be fully qualified:
```
<service in other namespace>.<its namespace>.svc.cluster.local
```

File: cqdg-orchestrations.iml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="WEB_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```
Comment (Magnitus-) [inline cqdg-orchestrations.iml]: Do we need this?

File: arranger/kustomization.yml [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+kind: Kustomization
+
+resources:
+  - ingresses.yml
+  - namespace.yml
+  - services.yml
+
+patchesStrategicMerge:
+  - configmaps.yml
+  - deployments.yml
```
Comment (Magnitus-) [inline arranger/kustomization.yml:8]: I think you want to put those two entries under "resources". "patchesStrategicMerge" is used when you want to "add to" and/or "overwrite" entries in a resource defined previously, but here, you are defining those resources for the first time.

File: arranger/ingresses.yml [hunk 6]
```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: arranger-server
+spec:
+  tls:
+    - hosts:
+        - arranger-server.qa.clin.ferlab.bio
```
Comment (Magnitus-) [inline arranger/ingresses.yml]: These domain names are very environment-specific. 

I would put generic entries here like this (but use "\_\_UNDEFINED\_\_", I started with "kustomizeMe" and then decided that "\_\_UNDEFINED\_\_" is more visible in the final output): https://github.com/Ferlab-Ste-Justine/cqdg-orchestrations/blob/master/keycloak/ingress.yml

btw, I think this name is missing from the extra entries in the certificate.

File: arranger/ingresses.yml [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+      secretName: letsencrypt-certificate
+  rules:
+    - host: arranger-server.qa.clin.ferlab.bio
```
Comment (Magnitus-) [inline arranger/ingresses.yml]: See comment above.

File: arranger/ingresses.yml [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+              servicePort: 5050
+---
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: arranger-admin-ui
+spec:
+  tls:
+    - hosts:
+        - arranger-admin.qa.clin.ferlab.bio
```
Comment (Magnitus-) [inline arranger/ingresses.yml]: See comment above.

File: arranger/ingresses.yml [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+      secretName: letsencrypt-certificate
+  rules:
+    - host: arranger-admin-ui.qa.clin.ferlab.bio
```
Comment (Magnitus-) [inline arranger/ingresses.yml]: See comment above.

File: cqdg-orchestrations.iml (hunk 4 above)
Comment (plaplante-lab) [reply cqdg-orchestrations.iml]: Nope, I'm gonna add a .gitignore

**Review by Magnitus- — APPROVED** [review]
LGTM

**Pull request merged**